### ISSUE TITLE
Zombie Awareness Crash (1.16.5)

### ISSUE THREAD

**Issue by Figgmastah** [BobMowzie/MowziesMobs#613]
When Mowzie's Mobs structures are loaded in with Zombie Awareness Mod on. The Game Crashes

**Comment by Figgmastah** [comment]
Yes, 100% Confirmed

**Comment by BobMowzie** [comment]
Okay, are you able to turn off certain Zombie Awareness features with their config? I saw this on  their Curseforge page `- Light sources: Zombies can see exposed light sources from super far away, and will come to investigate if they have line of sight` and I suspect this might be the issue.

**Comment by Figgmastah** [comment]
Will try, Weird that this would not work with Mowzie's Mobs. Would have been Big Bonus if Fixed. Posted this on Zombie Awareness mod too

**Comment by BobMowzie** [comment]
Yeah it’s very strange. The crash log just says the server froze, which doesn’t tell us much.
You could also try disabling my structures and seeing if that makes it go away. The Wroughtnaut chamber is the most unconventional structure, and could be the cause.

**Comment by Figgmastah** [comment]
I need The Structures and all of them Crashes The Game when trying to load

**Comment by BobMowzie** [comment]
I know you need the structures. We’re just trying to figure out which one causes the crash. The best way to do that is to test the game with them disabled.
My frostmaw structure and Barakoa village structure are implemented normally; if those also cause the crash, then it’s not a problem on my end.

**Comment by BobMowzie** [comment]
Which actually brings up another question. How can you be sure the crash happens when loading Mowzie’s Mobs structures?

**Comment by Figgmastah** [comment]
When I enter an area that one of your Structures are High Chance being the Game Crashes. Also tried to /Locate and Teleport to every single one of them. When done i can still move around and break blocks but they dont drop and nothing moves, only me. Until the game crashes

**Comment by BobMowzie** [comment]
Okay. Can you try teleporting to another mod’s structures too? I wonder if the same thing will happen.
Can you try spawning in my bosses manually without their structures and see if the server crashes?

**Comment by BobMowzie** [comment]
Also, this could be very useful if you can find a moment to try it: https://mcforge.readthedocs.io/en/1.14.x/gettingstarted/debugprofiler/

**Comment by Figgmastah** [comment]
Deactivating Light Source thing did not work. It didn't crash with other modded structures and every mob from your mod doesn't crash the game its the terrain generation. Will try that very useful thing
